ambry-clustermap/src/main/java/com.github.ambry.clustermap/HardwareLayout.java

```diff
@@ -19,6 +19,7 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import javax.xml.crypto.Data;
```

Comment:
Is this required? Also, have you imported our coding style file in your IntelliJ editor?